coalib/results/TextPosition.py

```diff
@@ -33,6 +33,19 @@ def column(self):
         return self._column
 
     @classmethod
+    def calc_position(self, text, line, column):
+        """
+        Calculates position in a string from line number and column.
+        Uses \\n  as the newline character. Position starts from 0
+        """
+        lines = text.splitlines(keepends=True)
```

Comment:
aah i thought you meant that 58570fb has a better algo for this one :P , nvm got it!